lib/plugins/aws/package/compile/events/alb/index.test.js

```diff
@@ -79,4 +81,55 @@ describe('AwsCompileAlbEvents', () => {
       return awsCompileAlbEvents.hooks['package:compileEvents']();
     });
   });
+
+  describe('onUnauthenticatedRequest', () => {
```

Comment:
@medikoo I thought before I start writing a set of tests to migrate them to use `runServerless` I will check what would like to change in the example test that tests the change to `onUnauthenticatedRequest`. 

Do you expect to have an assertion to https://github.com/serverless/serverless/pull/7780/files#diff-f210397c5bba20a4235eaae4251556f9R131 ?

Or

Should the generated cloudformation be asserted as that is the final result? 

I would probably opt-out for the latter one.

Reply:
Ideally if we confirm on generated template. So something as: https://github.com/serverless/serverless/blob/332524dae73cb102c244d3b568ec880f9bc816aa/lib/plugins/aws/package/compile/events/httpApi/index.test.js#L66 which will validate that we have `"deny"` set at given property.

Still if for some reason you'll find troublesome getting right path to resource to be validated, let me know